src/Themes/BootstrapTheme/Views/Layout.liquid

```diff
@@ -7,7 +7,10 @@
 
   <head>
     <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900" rel="stylesheet">
-
+    
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/knockout/3.4.0/knockout-min.js"></script>
```

Comment:
You can add these scripts from your views.

Reply:
And delete `Layout.liquid`.